seleniumwire/proxy/proxy2.py

```diff
@@ -331,23 +356,25 @@ def request(self, method, url, body=None, headers=None, *, encode_chunked=False)
         super().request(method, url, body, headers=headers)
 
 
-class ProxyAwareHTTPSConnection(HTTPSConnection):
+class ProxyAwareHTTPSConnection(HTTPSConnection, ProxyAwareConnectionMixin):
     """A specialised HTTPSConnection that will transparently connect to a
     HTTP or SOCKS proxy server based on supplied proxy configuration.
     """
 
     def __init__(self, proxy_config, netloc, *args, **kwargs):
         self.proxy_config = proxy_config
+        self.netloc = netloc
         self.use_proxy = 'https' in proxy_config and netloc not in proxy_config.get('no_proxy', '')
 
-        if self.use_proxy and proxy_config['https'].scheme.startswith('http'):
```

Comment:
Although it looks unnecessary, we do need to check the `scheme` here because it might be `socks5` if a [SOCKS](https://github.com/wkeeling/selenium-wire#socks) proxy is being used - e.g. 

    'proxy': {
        'https': 'socks5://user:pass@192.168.10.100:8889','
    }